howl/platform/oc.lua

```diff
@@ -0,0 +1,129 @@
+--- OpenComputers's platform table
```

Comment:
Would it be possible to indent this with tabs? I know Lua is "meant" to be indented with 2 spaces, but it's consistent with the rest of the project.

Reply:
I'll go through and indent with tabs then. I just wasn't sure what you wanted for indentation.